creating_images/custom.adoc

```diff
@@ -38,3 +40,13 @@ the information needed to proceed with the build:
 |`*DOCKER_SOCKET*`
 |This variable specifies the path to docker socket, if exposing docker socket was enabled on BuildConfig.
 |===
+
+== Custom builder workflow
+Although custom builder image author has a great flexibility in defining the build
+process on its own, still he should follow a few required steps necessary to seamlessly
+run a build inside of OpenShift. The required steps for a custom builder image are following:
+
+. Read the link:../rest_api/openshift_v1.html#v1-build[Build] definition, which
+contains all the necessary information about input parameters for the build.
+. Run the build process.
+. Output the artifacts to the link:../rest_api/openshift_v1.html#v1-build[Build]'s output location if one is defined.
```

Comment:
this really needs to say "push the image to the output location if the build produces an image and if the output location is defined"

our output is not (currently) a generic destination.     Whether in the future we want to make it one, or we just say "if you want to output to an ftp server, define an env variable in your buildconfig that the custom builder can look at to know where to upload to" i don't particularly care.